R/runComparison.R

```diff
@@ -1042,7 +1044,9 @@ runComparison <- function(file.table,
                           parameters, 
                           output.directory, 
                           check.table = TRUE, 
-                          out.width = NULL) {
+                          out.width = NULL,
+                          saveResultTable = FALSE,
```

Comment:
Optional arguments to save the result table (useful for simulation studies).
If set to their default value, the original behavior is preserved.

Reply:
Done, see commits 55aa274 and 3c74a6b